### DISCUSSION TITLE
Orchard Harvest 2023

### DISCUSSION THREAD

**Discussion by viktoriamagyar** [OrchardCMS/OrchardCore#12724] [Q&A]
What is your opinion of the next Orchard Harvest taking place in the first quarter of 2023?
Let us know if you would like to join the organizing team.
https://forms.office.com/r/145YRnJSKT
Share your thoughts with us in the survey. We are curious about who we will meet in 2023! :)

**Comment by Piedone** [comment]
We're still gathering details, so if you haven't yet, don't hesitate to send in the survey!

BTW about 10% of the respondents are some kind of Nicks, so maybe we need a special table just for the Nicks? :D

**Comment by MikeAlhayek** [comment]
@viktoriamagyar @Piedone if [Las Vegas](https://www.visitlasvegas.com/) is an option for the event, in addition to myself, I think I have the right contact needed to felicitate the event. [Las Vegas](https://www.visitlasvegas.com/) has been hosting lots of tech events for big companies. You can ping me on Gitter if you have more questions or interested in my offer :)

**Reply by hishamco** [reply]
Hope to be on Hungary ;) I hope to meet Orchard Core & Lombiq guys there

**Reply by viktoriamagyar** [reply]
@MikeAlhayek Thank you for the offer! :) We will look into that and get back to you. Las Vegas sounds really good as well.

**Reply by viktoriamagyar** [reply]
@hishamco I can't promise Hungary to you :( Hungary got the least vote according to the survey.

**Reply by hishamco** [reply]
At least it will be fair to make such event happen on each continents not only America

**Reply by MikeAlhayek** [reply]
@hishamco maybe it's a good time to visit the US :) https://ye.usembassy.gov/visas/

**Reply by hishamco** [reply]
USA is far away, it might needs a lot of money just to reach there - if it's possible for me-

**Reply by viktoriamagyar** [reply]
@MikeAlhayek I wrote you an email. Looking forward to your answer! :)

**Reply by Piedone** [reply]
I've unmarked this as answer just so the it doesn't give the false impression that this discussion is closed.

**Comment by hishamco** [comment]
BTW how many times the event was in USA? What you are opinion if we can made it in Asia? @viktoriamagyar what are the requirements to host such event may be I can use my contact to make it happen in KSA if it's possible

**Reply by hishamco** [reply]
@viktoriamagyar I contacted one of my friend in KSA to host such event, he welcomes the idea, could you please let me know what's the responsibilities to host such event in details?

**Comment by viktoriamagyar** [comment]
Dear @hishamco!
I have consulted with Zoltán on this. Thank you very much for your efforts and for asking, but at the moment KSA is not one of the target countries. There's very little Orchard community in the Middle East, that's why the survey was about US and Europe.
If you may have a friend in the target countries you can find every detail here: https://github.com/OrchardCMS/OrchardCore/discussions/13100

**Comment by agriffard** [comment]
Little list of steps related to the organization, based on previous Harvests:

Timeline from now :
- Contact local .NET users/companies.
- Contact hotels to check if we can have deals for bookings.
- Find potential sponsors.
 
3-4 months before:
- Confirm Location and Dates.
    People need to be able to decide/ask if they can come (convince their boss) a few months before the event.
- Call for speakers
             Nb sessions, duration ?
             Approvals
             Gather bios and pictures
 
- Communicate about the event.
- Website
             Sessions planning
             Speakers
             Sponsors
             Accommodations, Venue
 
- Event management
             Book tickets (a service like EventBrite has been used for previous Harvests)
             Maximum number of attendees?
             Price (cheaper if booked early?)
- Catering (small snacks)
             Ask for diet restrictions?
 
Day before:
- Print Badges to prepare Checkin.
 
Day 0:
- Welcome participants
- Give badges
 
Evening:
- Book a venue to take a drink between the attendees (and potential +1s).
 
Questions:
- Website Hosting           
- Domain name or subdomain
- Does the room have tables and power plugs?
- Do we record the sessions?
- Do we make T-shirts?
             - Logo, ...
             - Ask size during booking
             - Traveling with package

**Reply by Piedone** [reply]
Thank you for all of these, Antoine!

**Comment by viktoriamagyar** [comment]
Hello!

After trying for the last few weeks, we finally decided to hold the Orchard Harvest conference in Las Vegas in 2023 with the help of  @MikeAlhayek  . Of course, we are also curious to hear your opinion. Please let us know!
In my opinion, many would assume it's overpriced compared to most of the US, but Mike has assured me that our proposed date is one of the most affordable. Also some interesting sites on this:
https://www.elysianliving.com/news/cost-of-living-in-las-vegas-vs-your-state-ultimate-guide
https://livingcost.org/cost/las-vegas/new-york

What would be most important at the moment is to decide on a date. As I wrote before, it would definitely be better to have it before the summer break, so I would prefer to focus on May. On Sébastien's advice, I looked at previous Harvests, they were usually all on Monday and Tuesday, so the following dates would be a good choice:

- 8-9 May
- 15-16 May
- 22-23 May

Please let us know what you think! Who can we expect to see at the event?

Another important thing is that I'm going to put all the important news under this post so that you can follow it if you want to be informed about the news.
As some of the news has already been posted elsewhere, I'll link it here:
Orchard Harvest survey results: https://github.com/OrchardCMS/OrchardCore/discussions/13100
Orchard Harvest 2023: https://github.com/OrchardCMS/OrchardCore/discussions/13014

Looking forward to your opinions!

**Reply by Piedone** [reply]
How about a Twitter poll under the Orchard account? For me all are good BTW.

**Reply by hishamco** [reply]
Totally agree, looking forward ...

**Reply by MikeAlhayek** [reply]
Heads up May 14 is Mother’s Day! 8-9 May seems perfect to me

**Reply by viktoriamagyar** [reply]
@Piedone that would be great.
@MikeAlhayek thanks for the note. 
I would say let's have a Twitter poll under the Orchard account about whether to have it 8-9 May or 22-23 May.

**Reply by sebastienros** [reply]
I can send a poll on the twitter account

**Reply by Piedone** [reply]
May is also the month of Build, and [it may be on 22-23](https://www.windowscentral.com/microsoft/it-looks-like-the-date-for-microsoft-build-2023-has-just-leaked).

**Reply by viktoriamagyar** [reply]
https://twitter.com/orchardcms/status/1623784624605523968 The poll is out, please give a feedback! :)

**Reply by scleaver** [reply]
Not super keen on Las Vegas (but you can't keep everyone happy) there but I'll consider it further.

**Reply by viktoriamagyar** [reply]
@scleaver I am sorry to hear that. I hope you will be able to join us! :)

**Reply by hishamco** [reply]
@viktoriamagyar is the event will be streamming for those who unable to attend in person?

**Reply by viktoriamagyar** [reply]
It seems that the choosen date is the 22nd and 23rd of May! :) Hope most of you are happy with this date.
Next is finding the best location in Las Vegas. I am on it with the help of Mike. I will update you here as well.

Also, we are looking for sponsors and contributors.:

    - T-shirts
    - Location
    - a meet-up after the programs
    - video/ recording
    - renting equipment for presentations,
    - Catering
    - Webpage
If anyone could help in any way that would be really appreciated.

**Reply by hishamco** [reply]
What kind of help you need on the last point?

**Reply by viktoriamagyar** [reply]
Streaming is still a question, but in my opinion, recording will surely happen! I will make sure to update you about it.
Sebastian mentioned that in the past, every year someone would create a website to promote Orchard Harvest, thus supporting the event. So we are looking for someone who can help with that as well :)

**Reply by hishamco** [reply]
If you have a logo and all related thing I might help you on this :)

**Reply by Piedone** [reply]
We can host the site on DotNest for free.

**Reply by hishamco** [reply]
That's a good idea Zoltan, if @viktoriamagyar can provides me with all the required information, I might ask my friend which is a web design to make a website for the event

**Reply by Piedone** [reply]
Build is confirmed for 22-25 May: https://build.microsoft.com/en-US/home

**Comment by viktoriamagyar** [comment]
A brief summary of why it could be cheaper to hold the conference in Las Vegas:
Las Vegas hotels advertise cheap rooms in order to attract visitors to the city. Hotels can afford to keep rooms cheap because they make most of their money from other sources-like casinos, shows, and nightclubs. Las Vegas is heavily reliant on tourism to drive its economy. Las Vegas city make deals with airlines in order to lower airfare cost for travels. 
Las Vegas is heavily reliant on tourism to drive its economy. That is why hotel rooms and flights are so cheap. Another way to save a ton of money in Las Vegas is by booking your vacation during the week. Hotel rates are often 50% cheaper Sunday-Thursday than they are on Friday and Saturday.

**Comment by agriffard** [comment]
https://sessionize.com/ could be an interesting tool to manage the call for speakers.

**Reply by viktoriamagyar** [reply]
Thank you! I have saved it.

**Comment by agriffard** [comment]
If it can help or, at leave, give some design ideas, I have a license for this kind of template: https://www.okler.net/previews/porto/9.9.1/demo-event.html

**Comment by Piedone** [comment]
We have the date, the city, and most possibly we also have the venue, so we need to start publicizing the event ASAP. I see the following to be done for this:

- Create a web presence. This can be a standalone website (like harvest.orchardcore.net), or just a section under orchardcore.net. The goal is to have something up ASAP. Antoine [here](https://github.com/OrchardCMS/OrchardCore/discussions/12724#discussioncomment-5018823) mentioned an available template, and Hisham told he can work on the site. @hishamco are you still up for this?
- Publish announcements everywhere we can. Apart from the usual Orchard media, where can this be?
  - @sebastienros can you write a blog post for [the .NET Blog](https://devblogs.microsoft.com/dotnet/category/orchard-core/)?
  - Can .NET Foundation add a post as news [on their website](https://old.dotnetfoundation.org/blog) and/or added as [a news/event](https://dotnetfoundation.org/news-events/latest-news), and that be sent out in their newsletter?
  - Submit to get listed under [.NET Conferences and Events](https://dotnet.microsoft.com/en-us/platform/community/events).
  - Tweet to [@dotnet](https://twitter.com/dotnet).
  - Announce in the [.NET Developers LinkedIn group](https://www.linkedin.com/groups/40949/).
  - Announce in [r/dotnet](https://www.reddit.com/r/dotnet/).
  - Announce on the [C# Discord](https://aka.ms/csharp-discord).
  - Announce in the [.NET Community Hub](https://techcommunity.microsoft.com/t5/net/ct-p/dotnet?WT.mc_id=dotnet-35129-website).
  - Announce on [The .NET Platform Facebook page](https://www.facebook.com/Dotnet/).
- Evangelize Orchard on [.NET meetups around the world](https://www.meetup.com/pro/dotnet/), especially in North America (due to the vicinity), even if remotely, and publicize that we'll have a conference. Also via [.NET Virtual User Group](https://www.meetup.com/dotnet-virtual-user-group/).

**Reply by hishamco** [reply]
@agriffard can you share the theme via gitter or email it to my Hotmail, so I can check and see if there's a need to web designer

@Piedone I might email you after that to check who can I contact to get the event info

**Reply by hishamco** [reply]
@Piedone please check your email, I sent an email on 16-3 about the website

**Comment by viktoriamagyar** [comment]
Please check out my latest post: https://github.com/OrchardCMS/OrchardCore/discussions/13574
This year we will have a special online Orchard Harvest starting **on the 31st of May at 13:00 UTC**!

**Comment by Piedone** [comment]
We have our tentative program published! https://orchardcore.net/harvest

**Reply by MikeAlhayek** [reply]
@Piedone i don’t think I see the date posted anywhere on this link. Can we post Wed, May 31 at 5:00 AM - 1:00 PM PDT on there?

**Reply by hishamco** [reply]
![Harvest](https://github.com/OrchardCMS/OrchardCore/assets/3237266/a4c352f2-1a02-4c03-9142-7dac2ca5ddb2)

**Reply by Piedone** [reply]
I also added a World Time Buddy link that you can check for conversions: https://www.worldtimebuddy.com/?pl=1&lid=100,5368361,5419384,4887398,5128581,2643743,2988507,108410,1273294,1816670&h=100&date=5/31/2023%7C3&hf=1

**Comment by hishamco** [comment]
I think we need to remove the `Submit your talk for Orchard Harvest 2023` button with the above paragraph, also we can align other button to the left, seems they're not look properly

**Reply by Piedone** [reply]
Thanks, removed the talk button and text. The buttons are in the center deliberately :).

**Comment by Piedone** [comment]
Please share the word on Twitter (https://twitter.com/orchardcms/status/1661016194177204225) and LinkedIn (https://www.linkedin.com/feed/update/urn:li:share:7066781876387987456/)!

**Comment by Skrypt** [comment]
Guys, just want to share that I received the invitation from @viktoriamagyar about if I will attend/talk or not for the event. Due to the fact that I'm working on that day, I would be surprised to be able. I will surely try to watch some videos after 5 pm Eastern time though.

**Comment by Piedone** [comment]
Well, that was a blast! Thank you everyone for taking part!

**Comment by agriffard** [comment]
Thank you everyone one for the organization and the sessions.

**Comment by Piedone** [comment]
Just to share if somebody didn't see; here are the recordings of the sessions: https://www.youtube.com/playlist?list=PLpCsCyd254Fr-2gbJEOLAJaoU_4EqHKaI
